### ISSUE TITLE
[Pending Leave Request] didn't show information

### ISSUE THREAD

**Issue by huynq2007** [bitnami/bitnami-docker-orangehrm#35]
Hi there,
[Pending Leave Requests] didn't show requested leaves information even I can confirm it from leave list. Is it a bug or do I need to configure something?
<img width="942" alt="screen shot 2019-02-28 at 20 58 49" src="https://user-images.githubusercontent.com/39758800/53565495-0289cb80-3b9d-11e9-911a-031f8c86baff.png">

**Comment by juan131** [comment]
Hi @huynq2007 

That looks like an issue in the application (OrangeHRM) itself. Please note that we package OrangeHRM (along with its dependencies) on a Docker image and we maintain the image but not the application itself.

Could you please report this issue at https://github.com/orangehrm/orangehrm/issues? OrangeHRM developers might have more information about the reason why the "Pending Leave Requests" is empty.

**Comment by huynq2007** [comment]
@juan131
Thank you for pointing out.

Could you please tell me which version of OrangeHRM package are you using? 
Where can I get that package? I would like to investigate this issue further.

**Comment by juan131** [comment]
Hi @huynq2007 

We download and package the latest stable version from: https://sourceforge.net/projects/orangehrm/files/stable/

Depending on the version of the container you're using, you'll be using a different OrangeHRM version. You can obtain the version packaged on your container as shown below:

```
$ docker ps
CONTAINER ID        IMAGE                      ...
f230d9785050        bitnami/orangehrm:latest   ...
$ docker exec f230d9785050 /bin/bash -c 'echo $BITNAMI_IMAGE_VERSION'
4.2.0-1-debian-9-r88
```

**Comment by huynq2007** [comment]
Thanks @juan131

**Comment by juan131** [comment]
You're welcome @huynq2007 

Please let us know whether it's a known issue in OrangeHRM once the developers reply you.

**Comment by huynq2007** [comment]
@juan131 
It seems that this issue has been fixed in **OrangeRHM Version 4.3 Release**

Thank you for your supporting!

**Comment by juan131** [comment]
That's great @huynq2007 !! Thanks for notifying!
